src/demo/ui.jsx

```diff
@@ -714,15 +714,13 @@ class Pond extends React.Component {
     return (
       <Body onClick={this.onPondClick}>
         <Header>A.I. Results</Header>
-        {state.canSeePondText && <div style={styles.pondText}>
-          {pondText.map((text, index) => {
-            return (
-              <div>
-                {text}
-              </div>
-            );
-          })}
-        </div>}
+        {state.canSeePondText && (
+          <div style={styles.pondText}>
+            {pondText.map((text, index) => {
+              return <div key={index}>{text}</div>;
```

Comment:
note because this diff looks weird: i added `key={index}` here because React requires it

Reply:
Same :)
https://github.com/code-dot-org/ml-activities/pull/152/files#diff-0dfa01a8a25303d3430c725b8ffc48ceR722-R728